core/primitives/res/runtime_configs/52.json

```diff
@@ -0,0 +1,193 @@
+            "storage_write_value_byte": 31018539,
+            "storage_write_evicted_byte": 32117307,
+            "storage_read_base": 56356845750,
+            "storage_read_key_byte": 30952533,
+            "storage_read_value_byte": 5611005,
+            "storage_remove_base": 53473030500,
+            "storage_remove_key_byte": 38220384,
+            "storage_remove_ret_value_byte": 11531556,
+            "storage_has_key_base": 54039896625,
+            "storage_has_key_byte": 30790845,
+            "storage_iter_create_prefix_base": 0,
+            "storage_iter_create_prefix_byte": 0,
+            "storage_iter_create_range_base": 0,
+            "storage_iter_create_from_byte": 0,
+            "storage_iter_create_to_byte": 0,
+            "storage_iter_next_base": 0,
+            "storage_iter_next_key_byte": 0,
+            "storage_iter_next_value_byte": 0,
+            "touching_trie_node": 16101955926,
+            "promise_and_base": 1465013400,
+            "promise_and_per_promise": 5452176,
+            "promise_return": 560152386,
+            "validator_stake_base": 911834726400,
+            "validator_total_stake_base": 911834726400,
+            "alt_bn128_g1_multiexp_base": 713006929500,
+            "alt_bn128_g1_multiexp_byte": 3335092461,
+            "alt_bn128_g1_multiexp_sublinear": 4325094,
+            "alt_bn128_pairing_check_base": 9685508901000,
+            "alt_bn128_pairing_check_byte": 26575188546,
+            "alt_bn128_g1_sum_base": 3175314375,
+            "alt_bn128_g1_sum_byte": 76218543
+        },
+        "grow_mem_cost": 1,
+        "regular_op_cost": 822756,
+        "limit_config": {
+            "max_gas_burnt": 300000000000000,
+            "max_gas_burnt_view": 300000000000000,
+            "max_stack_height": 16384,
+            "stack_limiter_version": 1,
+            "initial_memory_pages": 1024,
+            "max_memory_pages": 2048,
+            "registers_memory_limit": 1073741824,
+            "max_register_size": 104857600,
+            "max_number_registers": 100,
+            "max_number_logs": 100,
+            "max_total_log_length": 16384,
+            "max_total_prepaid_gas": 300000000000000,
+            "max_actions_per_receipt": 100,
+            "max_number_bytes_method_names": 2000,
+            "max_length_method_name": 256,
+            "max_arguments_length": 4194304,
+            "max_length_returned_data": 4194304,
+            "max_contract_size": 4194304,
+            "max_transaction_size": 4194304,
+            "max_length_storage_key": 4194304,
+            "max_length_storage_value": 4194304,
+            "max_promises_per_function_call_action": 1024,
+            "max_number_input_data_dependencies": 128,
+            "max_functions_number_per_contract": 10000
+        }
+    },
+    "account_creation_config": {
+        "min_allowed_top_level_account_length": 32,
+        "registrar_account_id": "registrar"
+    }
+}
```

Comment:
Please minimize the diff with the previous config:

```
$ diff core/primitives/res/runtime_configs/52.json core/primitives/res/runtime_configs/50.json | wc -l
386
```

Reply:
(alternatively, prepend a commint which reformats previous configs to 4 spaces)

Reply:
good point - fixed.